config/config.go

```diff
@@ -31,7 +32,10 @@ import (
 
 // UseNewConfig controls whether to use the new TOML config format.
 // This is an experimental feature.
-var UseNewConfig = false
+var UseNewConfig = true
+
+// CurrentConfigVersion holds the current version of the config file
+const CurrentConfigVersion = "1.0"
```

Comment:
What does this version actually mean for future btw? Are incremental changes - i.e adding new fields = move to 1.1, and breaking changes would make it 2.0?

Reply:
I can't really see a scenario where it would Break, unless we change to a new format again.
I was thinking 1.1 for field addition/removal and 2.0 for new sections or section reorg.

Reply:
Ok, I agree with that but move removal to 2.0 cause that would technically be a "reorg" or breaking change. Thanks for clarifying!